actions/form/class.SimpleProperty.php

```diff
@@ -280,6 +279,24 @@ private function disableValues(core_kernel_classes_Property $property, Validator
         $element->setDisabledValues(array_unique($disabledValues));
     }
 
+    private function getSchemaProperties(): array
+    {
+        return array_merge(
+            tao_helpers_form_GenerisFormFactory::getDefaultProperties(),
+            array_filter(
+                [
+                    $this->getProperty(GenerisRdf::PROPERTY_ALIAS),
+                    $this->getProperty(GenerisRdf::PROPERTY_IS_LG_DEPENDENT),
+                    $this->getFeatureFlagChecker()->isEnabled('FEATURE_FLAG_STATISTIC_METADATA_IMPORT')
+                        ? $this->getProperty(GenerisRdf::PROPERTY_IS_STATISTICAL)
+                        : null,
+                    $this->getProperty(TaoOntology::PROPERTY_GUI_ORDER),
+                    $this->getProperty(ValidationRuleRegistry::PROPERTY_VALIDATION_RULE)
+                ]
+            )
+        );
```

Comment:
Suggestion/idea: As we are already doing a function call here to `array_merge`, we may put the conditional outside the main array and save the additional call to `array_filter`.

```suggestion
        return array_merge(
            tao_helpers_form_GenerisFormFactory::getDefaultProperties(),
            [
                $this->getProperty(GenerisRdf::PROPERTY_ALIAS),
                $this->getProperty(GenerisRdf::PROPERTY_IS_LG_DEPENDENT),                    
                $this->getProperty(TaoOntology::PROPERTY_GUI_ORDER),
                $this->getProperty(ValidationRuleRegistry::PROPERTY_VALIDATION_RULE)
            ],
            $this->getFeatureFlagChecker()->isEnabled('FEATURE_FLAG_STATISTIC_METADATA_IMPORT')
                        ? $this->getProperty(GenerisRdf::PROPERTY_IS_STATISTICAL)
                        : [],
        );
```

Reply:
I cant, cause I need the new field to be just after `PROPERTY_IS_LG_DEPENDENT`  and I do not want to overcomplicate and create index with order for the fields, etc.

Reply:
I also not want to split in 3 arrays to avoid using array_filter. I think the cost of having array_filter is not hight compared to visibility/readability.

Reply:
Ok then, I did not thought about the field order :facepalm:

Reply:
Thank you for the suggestion. I was also thinking about your proposal, but turned out array_filter was the clear way I found. If you have other idea, please share :)